staff/aurora-perez/precourse/bingo/index.js

```diff
@@ -0,0 +1,255 @@
+
+function finJuego(){ 
+    alert('Hasta la próxima!');
+    return
+};
+function bingo (){ 
+
+    //carton: array de objetos, con numero y match como propiedades
+    var carton =[
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       //linea2
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       //linea3
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+       { numero: 0, match: false},
+
+    ];
+    //contadores de lineas y bingo
+
+
+    var lineaCantada1 = false;
+    var lineaCantada2 = false;
+    var lineaCantada3 = false;
+    var contadorTurnos= 0
+
+
+    //generar array con numeros del 1 al 90 para el carton, de ahi iremos sacando numeros random que iran en el carton
+    var listaNumeros=[];
+    for (var i=1; i<91; i++){
+        listaNumeros.push(i)
+    };
+
+    
+    //generar array con numeros del 1 al 90 para el bombo, de ahi iremos sacando numeros random que iran en el bombo
+    var listaNumeros2 = [];
+    for (var i=1; i<91; i++){
+     listaNumeros2.push(i)
+    };
+
+
+
+    //generar lista vacia para poder generar numeros random
+    var numcarton=[];
+    function generarCarton () {
+        while(numcarton.length<15){
+            var temp = Math.floor(Math.random()*89)+1;

```

Comment:
Hi, you can Math.ceil instead of Math.floor and you can delete +1

Reply:
I didn't know Math.ceil before! Thanks !!:)